### DISCUSSION TITLE
Marta - File Manager for macOS (https://marta.sh/)

### DISCUSSION THREAD

**Discussion by garderobier** [catppuccin/catppuccin#2545] [Port Requests]
### Will you make the port?

- [ ] Yes, I'm willing to make the port

### How can the suggested port be themed?

https://marta.sh/docs/configuration/themes/

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by sgoudham** [comment]
Closing since https://github.com/catppuccin/catppuccin/issues/2732 has been created and in-review.
